packages/account/src/Components/api-token/api-token-clipboard.jsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIsMounted } from '@deriv/shared';
+import { Dialog, Icon, Text, Popover } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+
+const WarningNoteBullet = ({ message }) => (
+    <div className='da-api-token__bullet-wrapper'>
+        <div className='da-api-token__bullet' />
+        <Text as='p' color='prominent ' size='xs' line_height='m'>
+            {message}
+        </Text>
+    </div>
+);
+
+const WarningDialogMessage = () => (
+    <>
+        <Text as='p' color='prominent ' size='xs' line_height='m'>
+            {localize(
+                'Be careful who you share this token with. Anyone with this token can perform the following actions on your account behalf'
+            )}
+        </Text>
+        <div className='da-api-token__bullet-container'>
+            <WarningNoteBullet message={localize('Add accounts')} />
+            <WarningNoteBullet message={localize('Create or delete API tokens for trading and withdrawals')} />
+            <WarningNoteBullet message={localize('Modify account settings')} />
+        </div>
+    </>
+);
+
+const ApiTokenClipboard = ({
+    scopes,
+    text_copy,
+    info_message,
+    success_message,
+    popoverAlignment = 'bottom',
+    disableApp,
+    enableApp,
+}) => {
+    const [is_copied, setIsCopied] = React.useState(false);
+    const [is_visible, setIsVisible] = React.useState(false);
+    const [is_popover_open, setIsPopoverOpen] = React.useState(false);
+    const isMounted = useIsMounted();
+    let timeout_clipboard = null;
+    const has_admin_scope = scopes.includes('Admin');
+
+    const toggleDialogVisibility = event => {
+        setIsVisible(!is_visible);
+        event.stopPropagation();
+    };
+
+    const togglePopupvisibility = () => {
+        setIsPopoverOpen(!is_popover_open);
+    };
+
+    const copyToClipboard = text => {
+        const textField = document.createElement('textarea');
+        textField.innerText = text;
+        document.body.appendChild(textField);
+        textField.select();
+        document.execCommand('copy');
+        textField.remove();
+    };
+
+    const onClick = () => {
```

Comment:
clicking on `Ok` button results in removing the dialog without any animation, don't we need to have here?

Reply:
LGTM :+1: